File: SemanaUNO/SemanaUNO/Ejercicio5.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SemanaUNO
{
    internal class Ejercicio5
    {
        public void Calculadora() { 
        Console.Clear();

            int num1, num2, resultado;


            Console.WriteLine("Ingrese el primer número:");
            num1 = Convert.ToInt32(Console.ReadLine());

            Console.WriteLine("Ingrese el segundo número:");

            num2 = Convert.ToInt32(Console.ReadLine());
            resultado = num1 + num2;
            Console.WriteLine("La suma de " + num1 + " y " + num2 + " es: " + resultado);

            Console.ReadKey();
            Console.Clear();
        }
    }
}

File: SemanaUNO/SemanaTRES/Ejercicio5.vb
Public Class Ejercicio5


    Sub Main()

        Console.Clear()

        Dim numero As Integer

        Console.WriteLine("Ingrese un número:")
        numero = CInt(Console.ReadLine())

        If numero Mod 2 = 0 Then
            Console.WriteLine("El número " & numero & " es par.")

        Else
            Console.WriteLine("El número " & numero & " es impar.")
        End If

        Console.ReadKey()
        Console.Clear()

    End Sub

End Class
